web/client/components/data/grid/DataGrid.jsx

```diff
@@ -59,6 +59,8 @@ class DataGrid extends Grid {
             // force scroll top
             if (scrollToTopCounter !== oldProps.scrollToTopCounter && this.canvas) {
                 this.canvas.scrollTop = 0;
+            } else if (this.canvas) {
```

Comment:
Not sure, but if the header is not present, we may have a `Cannot read properties of undefined ` error, isn't it?
```suggestion
            } else if (this.canvas && this.header) {
```

Reply:
Hi Lorenzo!
it could be added as an additional check, although I believe, react-data-grid will always show a default header even if not specified, so I am not sure if the condition will play a roll at all.